inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -464,10 +464,8 @@ linelist_cleaned$age_group_mon <- age_categories(linelist_cleaned$age_months,
                                                  ceiling = TRUE)
 
 ## create an age group variable by specifying categorical breaks
-## categories are 0-5m, 6-11m, 12-23m, 24-59m, 5-14y, 15-29y, 30-44y, 45+
 linelist_cleaned$age_group <- age_categories(linelist_cleaned$age_years, 
-                                             breakers = c(0, 0.5, 1, 2, 5,
-                                                          15, 30, 45))
+                                             breakers = c(0, 5, 15, 30, 45))
```

Comment:
I changed the 5 yr age group to 3, this makes the lowest age category 0-2yrs which allows us to merge with the month variable above.

Reply:
also added breaks to conform with isidros requests of age-groups